plugins/genesis-tools/agents/agent-driver.md

```
@@ -0,0 +1,96 @@
+
+When the turn completes, run `VERIFY_CMD` **yourself** and read `git diff`. The worker's claim that it passed is not evidence. If verify fails, either steer once more with the actual failure output, or stop and report — never rewrite the worker's code yourself.
+
+## 8. Tear down and report
+
+```bash
+tools codex stop --name <NAME>
+```
+
+Report to `lead` in this shape, and nothing longer:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)